Cargo.toml

```diff
@@ -9,10 +9,15 @@ readme = "README.md"
 license = "Apache-2.0/MIT"
 categories = ["data-structures"]
 
+[features]
+bincode = ["dep:bincode", "vob/bincode", "packedvec/bincode"]
+serde = ["dep:serde", "vob/serde", "packedvec/serde"]
+
 [dependencies]
-vob = { version=">=3.0.2", features=["serde"] }
-packedvec = { version="1.0", features=["serde"] }
+vob = { version=">=3.0.4" }
```

Comment:
I made this one a draft primarily because I was wondering about this `"version = ">=3.0.4"`.
Perhaps I am blind, but I couldn't find anything in the cargo docs saying that this `>=` implies within semver range?

Reply:
Aha, I think `>=` means "this or later versions even with major changes". I think we should drop the `>=` so we are compatible with new minor, but not major, versions.

Reply:
Will gladly go ahead and remove it.

Reply:
Fixed in [ad58740](https://github.com/softdevteam/sparsevec/pull/28/commits/ad587405bef2b52138400db8256abffd26825f13) will probably want to squash.